.travis.yml

```diff
@@ -74,6 +74,9 @@ matrix:
     - if: NOT (type = pull_request)
       env:
       - TEST_TARGET="ci-coquelicot"
+    - if: NOT (type = pull_request)
+      env:
+      - TEST_TARGET="ci-elpi"
```

Comment:
If the opam packages in the other CI files are required they should be added to travis as well.